.github/workflows/build.yml

```diff
@@ -588,7 +678,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        script: [lua, moon, fennel, js, wren, squirrel]
+        script: [lua, moon, fennel, mruby, js, wren, squirrel]
```

Comment:
We have to rename it to `ruby` here or the `export ruby %name%` command won't work.

Reply:
Or, we can use `export mruby` as a variant.
Any thoughts?